src/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.cs

```diff
@@ -103,6 +103,7 @@ public enum EventKeywords : long
         Sqm = (long)2251799813685248,
         WdiContext = (long)562949953421312,
         WdiDiagnostic = (long)1125899906842624,
+        MicrosoftTelemetry = (long)562949953421312,
```

Comment:
Should be pretty simple tests right? APIs added are mostly enum fields, constructors that only initialize fields and simple properties.